### PR TITLE
Updated card component to HTML5

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW
This replaces `<div>` with `<article>` and `<section>`

# Summary
- Update Decanter to use HTML5

# Needed By (Date)
- soon

# Urgency
- Not urgent

# Steps to Test

1. Checkout this branch
2. Verify the code changes make sense
3. Verify the card component still works
4. Verify the rendered output (inspect) makes sense

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?
Decanter
# Associated Issues and/or People
- JIRA ticket - https://stanfordits.atlassian.net/browse/THEME-15
- Other PRs - https://github.com/SU-SWS/decanter/pull/421
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: core/src/templates/components/card/card.twig [hunk 1]
```diff
@@ -85,7 +85,7 @@
 
     {# Big ol button. Adds a BEM class for targeting. #}
     {%- if card_button_label is not empty and allow_links %}
-      <div class="su-card__button">
+      <button class="su-card__button">
```
Comment (sherakama) [inline core/src/templates/components/card/card.twig]: @cjwest This cannot be a `<button>` as it contains an `<a>` tag.

Comment (cjwest) [reply core/src/templates/components/card/card.twig]: Hah!

> Be careful when marking up links with the button role. Buttons are expected to be triggered using the Space key, while links are expected to be triggered through the Enter key.

**Comment by cjwest** [comment]
@sherakama,
Thank you for your comments. 
This is now ready for your review again!

**Comment by cjwest** [comment]
Duplicates work on https://github.com/SU-SWS/decanter/pull/474

**Pull request closed without merging**